### PR TITLE
Links for learning in the app opens in same page #249

### PR DESCRIPTION
**Pull request by Edu93Jer**
fix : Links for learning vulnerability now open in new tab,

Updated the [resources/i18n](https://github.com/SasanLabs/VulnerableApp/tree/master/src/main/resources/i18n) files and the [index.html](https://github.com/SasanLabs/VulnerableApp/blob/master/src/main/resources/static/index.html)

### CODE REVIEW

File: .settings/org.eclipse.buildship.core.prefs [hunk 1]
```diff
@@ -1,13 +1,2 @@
-arguments=
```
Comment (preetkaran20) [inline .settings/org.eclipse.buildship.core.prefs:1]: can we delete this file ?

File: src/main/resources/static/index.html [hunk 2]
```diff
@@ -31,7 +31,7 @@
 				Mode
 			</div>
 			<div class="navbar-item-menu">
-				<a href="https://github.com/SasanLabs/VulnerableApp"><img src="/images/GitHub-Mark-32px.png" /></a>
+				<a href="https://github.com/SasanLabs/VulnerableApp" target="_blank"><img src="/images/GitHub-Mark-32px.png" /></a>
```
Comment (preetkaran20) [inline src/main/resources/static/index.html:34]: Wow!!!

**Review by preetkaran20 — APPROVED** [review]

File: .settings/org.eclipse.buildship.core.prefs (hunk 1 above)
Comment (Edu93Jer) [reply .settings/org.eclipse.buildship.core.prefs:1]: Yes we can, sorry for the delay, it must be something of my IDE sorry about that.

**Comment by Edu93Jer** [comment]
Hi @preetkaran20  how are you, first of all, merry x-mas, just asking how is the review of the PR going, hope it is ok and it can be merged soon.
Cheers!

**Comment by preetkaran20** [comment]
LGTM, thanks a lot @Edu93Jer. Merging the PR.

**Pull request merged**